simapp/app.go

```diff
@@ -230,7 +230,7 @@ func NewSimApp(
 
 	// add keepers
 	app.AccountKeeper = authkeeper.NewAccountKeeper(
-		appCodec, keys[authtypes.StoreKey], app.GetSubspace(authtypes.ModuleName), sdk.Bech32MainPrefix, authtypes.ProtoBaseAccount, maccPerms,
+		appCodec, keys[authtypes.StoreKey], app.GetSubspace(authtypes.ModuleName), authtypes.ProtoBaseAccount, maccPerms,
```

Comment:
As we discussed it with @amaurym , we do not introduce any breaking changes in this PR and revert new argument in `NewAccountKeeper`

Reply:
Why? I think this is the type of breaking change that's fine. It's just app.go vs every module.

Reply:
We would like to introduce all breaking changes in the last PR (Step 3 from #9690)